File: AC/ABC-TDD/150/b.go
package main

import "fmt"

func main() {
	var n, cnt int
	var s string
	fmt.Scan(&n, &s)
	for i := 0; i < n-2; i++ {
		if string(s[i:i+3]) == "ABC" {
			cnt++
		}
	}
	fmt.Println(cnt)
}

File: AC/ABC-TDD/150/b.py
#n = int(input())
#s = input()
#cnt = 0
# for i in range(0, n-2):
#    if s[i:i+3] == "ABC":
#        cnt += 1
# print(cnt)

n = int(input())
s = input()
cnt = 0
for i in range(0, len(s)-2):
    if s[i] == "A" and s[i+1] == "B"and s[i+2] == "C":
        cnt += 1
print(cnt)
